redis/config.libsonnet

```diff
@@ -0,0 +1,31 @@
+{
+  _config+: {
+    namespace: error 'must specify namespace',
+    redis+: {
+      password: error 'must specify password',
+      port: 6379,
+      sentinel_port: 26379,
+      replicas: 3,
+      down_after_milliseconds: 15000,
+      diskSize: '50Gi',
```

Comment:
Is this a sane default? Seems huge to me for a start.

Reply:
It's opinionated, yes... but anyone can change it.